### PR TITLE
Replace dl-icon with clearly labeled button

### PR DESCRIPTION
**Pull request by trumanc**
https://github.com/usdigitalresponse/arpa-reporter/issues/300 mentions that users have reported not being able to find the download button. This PR replaces the nondescript download icon with a clear labeled action button that is much harder to miss.

| old | new |
--- | ---|
| Screen Shot 2022-07-04 at 2 23 40 PM | Screen Shot 2022-07-04 at 2 24 08 PM | 
Screen Shot 2022-07-04 at 2 23 47 PM | Screen Shot 2022-07-04 at 2 24 01 PM |

### CODE REVIEW

File: src/components/DownloadFileButton.vue [hunk 1]
```diff
@@ -1,18 +1,14 @@
 <template>
   <a :href="`/api/uploads/${upload.id}/download`">
-    <img
-      class="mr-4"
-      src="/assets/img/cloud-download.svg"
-      alt=""
-      width="20"
-      height="20"
-    />
+    <button class="btn btn-primary ml-2">
```
Comment (igor47) [inline src/components/DownloadFileButton.vue:3]: you don't need to embed a `button` inside an `a` anchor. instead, use button classes on the anchor itself. https://getbootstrap.com/docs/4.6/components/buttons/#button-tags

**Review by igor47 — APPROVED** [review]
lgtm, thanks, left a minor improvement suggestion

File: src/components/DownloadFileButton.vue (hunk 1 above)
Comment (trumanc) [reply src/components/DownloadFileButton.vue]: updated to inline the button class on the anchor tag, and also removed the now unused svg file

File: src/components/DownloadFileButton.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<template>
+  <a :href="`/api/uploads/${upload.id}/download`" class="btn btn-primary ml-2">
```
Comment (ajhyndman) [inline src/components/DownloadFileButton.vue:2]: You could try adding the class `btn-sm` as well.  Maybe that'll help prevent content from wrapping quite so easily?

**Review by ajhyndman — COMMENTED** [review]
I think it might actually be sufficient to swap to a more recognizable icon, but this is probably an improvement.  👍 

Material Icons:

image

**Comment by trumanc** [comment]
> I think it might actually be sufficient to swap to a more recognizable icon
Yeah I started with that idea but I figured it was better to make this button TOO visible instead of going with an icon that users might still miss. Better to fix this once instead of waiting for more user reports

Screen Shot 2022-07-05 at 11 29 28 AM

After making the button `btn-sm` the screen now looks like:

**Pull request merged**